tests/framework/types.go

```diff
@@ -93,6 +93,7 @@ type InstallOSMOpts struct {
 	CertmanagerIssuerGroup string
 	CertmanagerIssuerKind  string
 	CertmanagerIssuerName  string
+	CertBitSize            int
```

Comment:
super small nit: could we call with `CertKeyBitSize` for consistency?

Reply:
Good point! Done!